test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,42 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementisAnyNull() throws Exception {
```

Comment:
The test itself is correctly written,
But, is `elementisAnyNull` the name of the method you are testing?
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.
Remember that the first part of the name is the name of the method you are testing.